From: Robert Garrett <robertgarrett404@gmail.com>
Date: Wed, 9 Jul 2025 10:43:30 -0500
Subject: Always delete binary on clean, even if not present

Forcibly remove the binary during cleanup. If it's not present, I want
to quietly ignore that, rather than fail with a missing-file error.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index e2e2c61..a190781 100644
--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,7 @@ all: $(EXE)
 
 # The only build artefact at this time is the finished executable.
 clean:
-	rm $(EXE)
+	rm -f $(EXE)
 
 # Install the program into `/usr/bin/` according to the FHS
 # `DESTDIR` is normally empty and will have no effect. It's for future Debian
